src/components/LayoutTree.tsx

```diff
@@ -358,13 +358,26 @@ const LayoutTree = () => {
   const treeThoughts = useSelector(linearizeTree, _.isEqual)
   const fontSize = useSelector(state => state.fontSize)
   const dragInProgress = useSelector(state => state.dragInProgress)
+  const ref = useRef<HTMLDivElement | null>(null)
   const indentDepth = useSelector(state =>
     state.cursor && state.cursor.length > 2
       ? // when the cursor is on a leaf, the indention level should not change
         state.cursor.length - (hasChildren(state, head(state.cursor)) ? 2 : 3)
       : 0,
   )
 
+  const bullet = ref.current?.querySelector('[aria-label=bullet]')
```

Comment:
This needs to go in the `useLayoutEffect`. Never query the DOM in the body of a React function.